wondrous-app/components/Common/DeleteTaskModal/index.tsx

```diff
@@ -113,14 +113,14 @@ function DeleteTaskModal(props: IArchiveTaskModalProps) {
         <StyledCloseButton onClick={onClose}>
           <CloseModalIcon />
         </StyledCloseButton>
-        <StyledHeader>Delete this {taskType}?</StyledHeader>
```

Comment:
i think we should do an overall renaming. should be called DeleteEntityModal,
taskType -> `entityType`
taskTitle -> `entityTypeDisplayName`

also add comment about what entities this is used for